### PR TITLE
Homepage to load units by content language.

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -1359,13 +1359,22 @@ func handleCollection(db *sql.DB, r ItemRequest) (*Collection, *HttpError) {
...
+		inLangs = append(inLangs, fmt.Sprintf("'%s'", lang))
+	}
 	const queryTemplate = `
 WITH CUs AS (
 	SELECT ct.id as type_id, uid, cu_id AS id, film_date
 	FROM ( VALUES (%d), (%d), (%d), (%d), (%d), (%d), (%d) ) ct(id)
 	INNER JOIN LATERAL (
 		SELECT uid, id AS cu_id, coalesce(properties ->> 'film_date', created_at :: TEXT) :: DATE AS film_date
 		FROM content_units cu
+		INNER JOIN (
```
Comment (davgur) [inline api/handlers.go:1373]: is its work for you? 
Look like its condition for inner select and not for select from line 1371

**Review by davgur — APPROVED** [review]

File: api/handlers.go (hunk 1 above)
Comment (bbfsdev) [reply api/handlers.go:1373]: Yes it is inner join on content units.

Comment (bbfsdev) [reply api/handlers.go:1373]: I rolled this on staging backend.

Comment (bbfsdev) [reply api/handlers.go:1373]: Other selects are not units (are collections).

**Pull request merged**